source/AI.cpp

```diff
@@ -3437,7 +3441,30 @@ void AI::AimTurrets(const Ship &ship, FireCommand &command, bool opportunistic)
 				}
 
 				// Determine how much the turret must turn to face that vector.
-				double degrees = (Angle(p) - aim).Degrees();
+				double degrees = 0.;
+				Angle angleToPoint = Angle(p);
+				if(hardpoint.IsOmnidirectional())
+					degrees = (angleToPoint - aim).Degrees();
+				else
+				{
+					const Angle facing = ship.Facing();
```

Comment:
a comment on what comes after this line pls

Reply:
Done: [5a6fdbc](https://github.com/endless-sky/endless-sky/pull/7094/commits/5a6fdbc89c6a66333aba2b967ffbcca9a33fc01e)